thrift/src/main/thrift/atoms/quiz.thrift

```diff
@@ -62,11 +61,11 @@ struct QuizContent {
 struct QuizAtom {
   // do we need to store the ID, seeing as it is replicated(?) in the
   // content-atom wrapping?
-  1  : required string id
-  2  : required string title
-  7  : required bool published
-  6  : required bool revealAtEnd
-  8  : required string quizType
-  9  : optional i16 defaultColumns
-  10 : required QuizContent content
+  1: required string id
+  2: required string title
+  6: required bool revealAtEnd
```

Comment:
We should define our fields based on the logical ordering of their numbering. This shouldn't have any impact by switching the fields around as Thrift relies on the number to uniquely identify the field.